packages/mui/.storybook/preview.js

```diff
@@ -17,4 +17,4 @@ export const globalTypes = {
   },
 };
 
-export const decorators = [themeDecorator];
+export const decorators = [themeDecorator, calendarDecorator()];
```

Comment:
why ?

Reply:
Deleted there and added to all picker stories!